pycls/datasets/cifar10.py

```diff
@@ -48,13 +47,13 @@ def _load_data(self):
         inputs, labels = [], []
         for batch_name in batch_names:
             batch_path = os.path.join(self._data_path, batch_name)
-            with g_pathmgr.open(batch_path, "rb") as f:
+            with pathmgr.open(batch_path, "rb") as f:
                 data = pickle.load(f, encoding="bytes")
             inputs.append(data[b"data"])
             labels += data[b"labels"]
         # Combine and reshape the inputs
         inputs = np.vstack(inputs).astype(np.float32)
-        inputs = inputs.reshape((-1, 3, self._im_size, self._im_size))
+        inputs = inputs.reshape((-1, 3, cfg.TRAIN.IM_SIZE, cfg.TRAIN.IM_SIZE))
```

Comment:
This was done because the cfg was empty here when I ran it. We should verify that you're not just re-breaking something